great_expectations/experimental/datasources/interfaces.py

```diff
@@ -496,6 +498,7 @@ def head(
             HeadData
         """
         self.data.execution_engine.batch_manager.load_batch_list(batch_list=[self])
+        # TODO: <Alex>ALEX_NOTE-1/27/2023: This should be accomplished through calling "MetricsCalculator.head()"; however, circular-imports problem currently prevents importing "MetricsCalculator".</Alex>
```

Comment:
Is there a ticket to capture this? If not, let's do so because code comments tends to get lost in the mix

Reply:
@cdkini Good idea -- I will create it.  Thanks.

Reply:
@cdkini This has been done.  Thanks.